packages/utils/run-prettier.ts

```diff
@@ -11,26 +9,26 @@ const chalk = require("chalk");
  * @param {String} outputPath - Path to write the config to
  * @param {Node} source - AST to write at the given path
  * @param {Function} cb - executes a callback after execution if supplied
- * @returns {Function} Writes a file at given location and prints messages accordingly
+ * @returns {Void} Writes a file at given location and prints messages accordingly
  */
 
-module.exports = function runPrettier(outputPath, source, cb) {
-	function validateConfig() {
-		let prettySource;
-		let error;
+export default function runPrettier(outputPath: string, source: any, cb?: Function): void {
+	function validateConfig(): void | Function {
+		let prettySource: string;
+		let error: object;
 		try {
 			prettySource = prettier.format(source, {
 				singleQuote: true,
+				tabWidth: 1,
 				useTabs: true,
-				tabWidth: 1
 			});
 		} catch (err) {
 			process.stdout.write(
 				"\n" +
 					chalk.yellow(
 						`WARNING: Could not apply prettier to ${outputPath}` +
-							" due validation error, but the file has been created\n"
-					)
+							" due validation error, but the file has been created\n",
```

Comment:
no comma

Reply:
I personally prefer `trailing-comma` rule. Should I disable that rule?

Reply:
oh okay, no worries let's keep it